tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES6",
+    "target": "ES2017",
```

Comment:
why change the `target` from ES6 to ES2017?

Reply:
My fault. I will change it back.